.index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```

Comment:
On purpose. The new setup will create separate docs for nbd and nbd-lwt-unix, so the root of xapi-project.github.io/nbd is now a folder with two folders and no index. This is the index (See `.docgen.sh`)